src/main/java/io/excaliburfrc/robot/subsystems/Climber.java

```diff
@@ -94,6 +106,43 @@ public void closeAngler() {
     anglerPiston.set(DoubleSolenoid.Value.kReverse);
   }
 
+  private void toSetpoint(TrapezoidProfile.State setpoint) {
+    ElevatorFeedforward ff =
+        anglerPiston.get() == DoubleSolenoid.Value.kForward ? diagonalFF : upFF;
+    leftController.setReference(
+        setpoint.velocity,
+        ControlType.kVelocity,
+        0,
+        ff.calculate(setpoint.velocity),
+        ArbFFUnits.kVoltage);
+    rightController.setReference(
+        setpoint.velocity,
+        ControlType.kVelocity,
+        0,
+        ff.calculate(setpoint.velocity),
+        ArbFFUnits.kVoltage);
+  }
+
+  private void toSetpointWithDiagonal(TrapezoidProfile.State setpoint) {
+    if (setpoint.position >= HEIGHT_TO_OPEN_PISTON
+        && anglerPiston.get() != DoubleSolenoid.Value.kForward) {
+      anglerPiston.set(DoubleSolenoid.Value.kForward);
+    }
+    toSetpoint(setpoint);
+  }
+
+  private Command runForEverySetpoint(Consumer<TrapezoidProfile.State> toRun) {
```

Comment:
I'm not sure of the name for this method

Reply:
Any better names?